lib/themes/dosomething/paraneue_dosomething/scss/content/_finder.scss

```diff
@@ -213,7 +213,7 @@
   }
 
   .no-result {
-    background: $off-black asset-url("images/placeholder.png") center center;
+    background: $off-black center center;
```

Comment:
I think so, although it doesn't really make any sense to set background positioning now. :confused: